clients/command/src/test/java/org/eclipse/hono/client/command/CommandRouterCommandConsumerFactoryTest.java

```diff
@@ -38,25 +38,24 @@
 
 import io.vertx.core.Future;
 
-
 /**
- * Tests verifying behavior of {@link CommandRouterCommandConsumerFactory}.
+ * Tests verifying behavior of {@link ProtocolAdapterCommandConsumerFactoryImpl}.
  *
  */
 public class CommandRouterCommandConsumerFactoryTest {
```

Comment:
The `CommandRouterCommandConsumerFactoryTest` test class should be renamed as well.

Reply:
done